src/xtgeo/surface/_surfs_import.py

```diff
@@ -9,7 +9,7 @@
 logger = xtg.functionlogger(__name__)
 
 
-def from_grid3d(self, grid, subgrids, rfactor):
+def from_grid3d(grid, subgrids, rfactor):
     """Get surfaces from 3D grid, including subgrids"""
```

Comment:
Docstring seems outdated now

Reply:
Is it really? It might just be late in the day, but we are still getting surfaces from 3D grid?

Reply:
Slightly different as we are getting a triple and not an Surfaces object. The wording was already vague and not strictly correct as you would change the current object to the new contents.